requirements.txt

```diff
@@ -1,9 +1,14 @@
-click==8.1.3
 Flask==2.1.2
+Flask_Bcrypt==0.7.1
+Flask_Cors==3.0.10
+flask_mongoengine==1.0.0
+python-dotenv==0.20.0
+python_bcrypt==0.3.2
+Werkzeug==2.0.0
+zipp==3.8.0
+nodeenv==1.6.0
 importlib-metadata==4.11.3
 itsdangerous==2.1.2
-Jinja2==3.1.2
+Jinja2==3.0.3
 MarkupSafe==2.1.1
```

Comment:
Try not to update the version in requirement.txt. Version changes can mess up code in other places.